packages/cozy-sharing/src/state.js

```diff
@@ -420,3 +430,19 @@ const getAppUrl = (apps, appName) => {
   }
   return app.links.related
 }
+
+/**
+ *
+ * @param {SharingCollection} sharings
+ * @return {Array} Array of docIds
+ */
+export const getSharedDocIdsBySharings = sharings => {
+  const docs = []
+  if (!sharings.data) return []
+  sharings.data.map(s => {
```

Comment:
Nit, but that's a `forEach` rather than `map`.

Reply:
thanks :)